src/main/java/com/amalitech/gpuconfigurator/service/UserService.java

```diff
@@ -0,0 +1,62 @@
+package com.amalitech.gpuconfigurator.service;
```

Comment:
Have an Interface of your services, and create Implementation classes that Implement said interfaces. You should always strive to code to an interface and not a implementation.